index.html

```diff
@@ -3546,68 +3543,11 @@ <h3>Zero-Knowledge Proofs</h3>
-to secure a <a>verifiable credential</a> using a normatively defined method that
-enable zero knowledge proofs, or they will be removed.
-        </p>
-        <p>
-The following example shows one method of using <a>verifiable credentials</a> in
-zero-knowledge. It makes use of a Camenisch-Lysyanskaya Signature
-[[?CL-SIGNATURES]], which allows the presentation of the <a>verifiable
-credential</a> in a way that supports the privacy of the
-<a>holder</a> and <a>subject</a> through the use of selective disclosure of the
-<a>verifiable credential</a> values. Some other cryptographic systems which rely
-upon zero-knowledge proofs to selectively disclose attributes can be found in the
-Verifiable Credential Specifications Directory [[?VC-SPECS]] as well.
-        </p>
-
-        <pre class="example nohighlight" title="A verifiable credential that supports CL Signatures">
-{
-  "@context": [
-    "https://www.w3.org/ns/credentials/v2",
-    "https://www.w3.org/ns/credentials/examples/v2"
-  ],
-  "type": ["VerifiableCredential", "ExampleDegreeCredential"],
-  <span class="highlight">"credentialSchema": {
-    "id": "did:example:cdf:35LB7w9ueWbagPL94T9bMLtyXDj9pX5o",
-    "type": "did:example:schema:22KpkXgecryx9k7N6XN1QoN3gXwBkSU8SfyyYQG"
-  }</span>,
-  "issuer": "did:example:Wz4eUg7SetGfaUVCn8U9d62oDYrUJLuUtcy619",
-  "credentialSubject": {
-    "givenName": "Jane",
-    "familyName": "Doe",
-    "degree": {
-      "type": "ExampleBachelorDegree",
-      "name": "Bachelor of Science and Arts",
-      "college": "College of Engineering"
-    }
-  },
-  <span class="highlight">"proof": {
-    "type": "CLSignature2019",
-    "issuerData": "5NQ4TgzNfSQxoLzf2d5AV3JNiCdMaTgm...BXiX5UggB381QU7ZCgqWivUmy4D",
-    "attributes": "pPYmqDvwwWBDPNykXVrBtKdsJDeZUGFA...tTERiLqsZ5oxCoCSodPQaggkDJy",
-    "signature": "8eGWSiTiWtEA8WnBwX4T259STpxpRKuk...kpFnikqqSP3GMW7mVxC4chxFhVs",
-    "signatureCorrectnessProof": "SNQbW3u1QV5q89qhxA1xyVqFa6jCrKwv...dsRypyuGGK3RhhBUvH1tPEL8orH"
-  }</span>
-}
-        </pre>
-        <p>
-The example above provides the <a>verifiable credential</a> definition by using
-the <code>credentialSchema</code> <a>property</a> and a specific proof that is
-usable in the Camenisch-Lysyanskaya Zero-Knowledge Proof system.
-        </p>
-
-        <p>
-The next example utilizes the <a>verifiable credential</a> above to generate a
-new derived <a>verifiable credential</a> with a privacy-preserving proof. The
-derived <a>verifiable credential</a> is then placed in a
-<a>verifiable presentation</a>, so that the <a>verifiable credential</a>
-discloses only the <a>claims</a> and additional credential metadata that the
-<a>holder</a> intended. To do this, all of the following requirements are
-expected to be met:
+        <p class="issue">
+Examples of leveraging <a href="https://w3c.github.io/vc-di-bbs/">vc-di-bbs</a>,
+will be added here in the future, or this section will be removed.
```

Comment:
Maybe just leave this out until it's ready and open an issue instead?

If it stays here, though, it should get the "at-risk" label so it doesn't get overlooked during editing.

Reply:
IMO this current text accomplishes the same goal, but happy to apply a suggestion if you have one, and the group likes it.